cmd/mpi-operator.v1/app/options/options.go

```diff
@@ -23,15 +23,16 @@ import (
 
 // ServerOption is the main context object for the controller manager.
 type ServerOption struct {
-	Kubeconfig           string
-	MasterURL            string
-	KubectlDeliveryImage string
-	Threadiness          int
-	MonitoringPort       int
-	PrintVersion         bool
-	GangSchedulingName   string
-	Namespace            string
-	LockNamespace        string
+	Kubeconfig            string
+	MasterURL             string
+	KubectlDeliveryImage  string
+	Threadiness           int
+	MonitoringPort        int
+	PrintVersion          bool
+	GangSchedulingName    string
+	Namespace             string
+	LockNamespace         string
+	LauncherRunsWorkloads bool
```

Comment:
Maybe slightly better to use the singular form: `LauncherRunsWorkload`

Reply:
Thanks, i will fix it.